
Inherited from the older single-part patch, this part moved to its own
patch at 1.26.0 since it needed src changes.

--- mate-applets-1.26.1/multiload/src/linux-proc.c.orig
+++ mate-applets-1.26.1/multiload/src/linux-proc.c
@@ -384,6 +384,8 @@
     MultiloadApplet *multiload;
 
     multiload = g->multiload;
+
+    netlist.flags = 1;
     devices = glibtop_get_netlist(&netlist);
 
     for(i = 0; i < netlist.number; ++i)
